src/index.js

```diff
@@ -203,13 +221,19 @@ export function validate(config, metaConfig) {
 }
 
 function assertValid(value, validateKey, validator) {
-    if (!assert(value, validator)) {
-        throwError(validateKey);
+    const result = assert(value, validator);
```

Comment:
The behaviour is the same, no? It still checks if it is valid and throws if not.